dutyfree-chart/templates/certificate.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.certificate.enabled }}
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
```

Comment:
oh, that's quite nice!
didn't know about this 😁 

https://github.com/GoogleCloudPlatform/gke-managed-certs

Reply:
yeah, just discovered that recently, We don't have to manage anything. 😁